src/Database/Mysql/Driver/Mysql.php

```diff
@@ -57,7 +61,14 @@ public function create(Tenant $tenant): array
      */
     public function update(Tenant $tenant): array
     {
-        return [];
+        $config = $this->configure($tenant, true);
+
+        if(!isset($config['oldUsername'])){
```

Comment:
Return type is array. This won't work.

Reply:
Adjusted this so it returns an empty array, my bad.